x-pack/legacy/plugins/reporting/server/lib/get_user.ts

```diff
@@ -6,27 +6,14 @@
 
 import { Legacy } from 'kibana';
 import { KibanaRequest } from '../../../../../../src/core/server';
-import { Logger, ServerFacade } from '../../types';
+import { ServerFacade } from '../../types';
 import { ReportingSetupDeps } from '../plugin';
 
-export function getUserFactory(
-  server: ServerFacade,
-  security: ReportingSetupDeps['security'],
-  logger: Logger
-) {
+export function getUserFactory(server: ServerFacade, security: ReportingSetupDeps['security']) {
   /*
    * Legacy.Request because this is called from routing middleware
    */
   return async (request: Legacy.Request) => {
-    if (!security) {
-      return null;
-    }
-
-    try {
-      return await security.authc.getCurrentUser(KibanaRequest.from(request));
-    } catch (err) {
-      logger.error(err, ['getUser']);
-      return null;
-    }
+    return security?.authc.getCurrentUser(KibanaRequest.from(request)) ?? null;
```

Comment:
__note:__ these changes aren't mandatory, just forgot to remove this cleanup. For the majority of places I tried to not change anything.